analyzers/dataframe/MCParticle.cc

```diff
@@ -676,9 +683,20 @@ ROOT::VecOps::RVec<float> MCParticle::AngleBetweenTwoMCParticles( ROOT::VecOps::
   }
 
   return result;
+}
 
+ROOT::VecOps::RVec<edm4hep::MCParticleData> MCParticle::get(ROOT::VecOps::RVec<int> index, ROOT::VecOps::RVec<edm4hep::MCParticleData> in){
```

Comment:
I guess this is to get a list of MCParticle from a list of indices. The name get is then very confusing.

Reply:
I change it in a new commit, don't hesitate if you have any other suggestion for the name.